src/components/Hamburger/index.tsx

```diff
@@ -37,7 +37,7 @@ export default class Hamburger extends Component<HamburgerProps, HamburgerState>
             className="buttonExit"
             onClick={this.handleClick}>
           </button>
-          {window && window.location.pathname !== '/' && <Link to="/" onClick={this.handleClick}> Home </Link>}
+          {window && window.location.hash !== '#/' && <Link to="/" onClick={this.handleClick}> Home </Link>}
```

Comment:
Would be better not to depend on window here, as that stuff will change based on Hash/Browser history every time
Try
```
<Route exact path="/" render={() => <Link to="/" onClick={this.handleClick}> Home </Link>}/>
```

Reply:
but this doesn't make sense, render Home when on "/" aka home path? should be the inverse

Reply:
Yes, a million times
```
<Route path="/(.+)" render={() => <Link to="/" onClick={this.handleClick}> Home </Link>}/>
```
this works

Reply:
cool, good stuff ;)